src/app/phast/phast-report/report-graphs/report-graphs.component.html

```diff
@@ -7,9 +7,29 @@
       </select>
     </div>
     <div class="p-2 pie-chart-container">
-      <app-plotly-pie-chart [valuesAndLabels]="selectedBaselineData.valuesAndLabels" [valuesUnit]="lossUnit">
+      <app-plotly-pie-chart *ngIf="selectedBaselineData.valid.isValid" [valuesAndLabels]="selectedBaselineData.valuesAndLabels" [valuesUnit]="lossUnit">
       </app-plotly-pie-chart>
     </div>
+    <div class="alert-danger small" *ngIf="!selectedBaselineData.valid.isValid">
```

Comment:
This appears to be a repeated logic as other places. Should make a shared component in phast that takes the valid object and does this display logic. Didn't do that in psat/fsat because it only had 4 lines